root/etc/e-smith/templates.metadata/etc/nethserver/cockpit.allow

```diff
@@ -0,0 +1,3 @@
+UID="root"
+GID="root"
+PERMS="0440"
```

Comment:
Why such restrictive permissions? Is't the default root/root `rw-r--r--` ok?

I think this metadata file is not required.

Reply:
yes this one is on my own, maybe we could be less strict :-?

Reply:
done